listenbrainz/webserver/static/js/src/fresh-releases/ReleaseTimeline.tsx

```diff
@@ -31,14 +31,36 @@ export default function ReleaseTimeline(props: ReleaseTimelineProps) {
     setMaxSliderDate(new Date(dates[dates.length - 1]).getTime());
   }
 
-  const onDateChange = (newDate: any) => {
-    setCurrentValue(newDate);
+  function createMarks(data: FreshReleaseItem[]) {
+    const releasesPerDate = countBy(releases, (item) => item.release_date);
+    const datesArr = Object.keys(releasesPerDate).map((item) =>
+      formattedReleaseDate(item)
+    );
+    const percentArr = Object.values(releasesPerDate)
+      .map((item) => Math.floor((item / data.length) * 100))
+      .map((_, index, arr) =>
+        arr.slice(0, index + 1).reduce((prev, curr) => prev + curr)
+      );
+    percentArr.unshift(0);
+    percentArr.pop();
```

Comment:
`unshift` and `pop` explaination - we want the percentage values to start at 0 to get the first date at the start of the slider where the respective releases are. And then we wan to remove the last value.

Reply:
Might be good to move this comment from Github to inside the code.

Although if I'm honest I'm still not clear on why we're removing the first and last elements. Is that why there is no date for the end of the slider?